docker/pipelines_runner.sh

```diff
@@ -73,13 +85,26 @@ function main {
     exit 1
   fi
 
+  # Build up the extra args is they are provided
+  if [[ ! -z "${subnetwork}" ]]; then
+    echo "Adding --subnetwork ${subnetwork} to optional_args"
+    pt_optional_args="${pt_optional_args} --subnetwork projects/${project}/regions/${region}/subnetworks/${subnetwork}"
+    df_optional_args="${df_optional_args} --subnetwork https://www.googleapis.com/compute/v1/projects/${project}/regions/${region}/subnetworks/${subnetwork}"
+  fi
+
+  if [[ ! -z "${use_public_ips}"  && "${use_public_ips}" == "true" ]]; then
+    echo "Adding --private-address and --no_use_public_ips to optional_args"
```

Comment:
s/private-address/privateAddress/

Reply:
The external flag for the pipelines tool is `private-address` but the var name is `privateAddress`.

https://github.com/googlegenomics/pipelines-tools/blob/master/pipelines/internal/commands/run/run.go#L159

Reply:
my bad, thanks for correcting me.